src/obr/signac_wrapper/operations.py

```diff
@@ -596,12 +603,19 @@ def run_cmd_builder(job: Job, cmd_format: str, args: dict) -> str:
     return cmd_format.format(**cli_args) + "|| true" + postflight_cmd
 
 
+def validate_state(_: str, job: Job) -> str:
+    """Perform a detailed update of the job state"""
+    case = OpenFOAMCase(str(job.path) + "/case", job)
```

Comment:
Isn't `job.path` a `Path`? I.e. wouldn't `job.path \ "case"` work?

Reply:
No, it is a string. But in any case`str(job.path)` is not needed. In the other cases we convert `job.path` to a Path, which is also somewhat dubious because that happens in the constructor of OpenFOAMCase anyways.